### PR TITLE
Minimal N-EUREKA regression test.

### PR DESCRIPTION
**Pull request by FrancescoConti**

### CODE REVIEW

**Comment by ricted98** [comment]
@yvantor I think we should merge it to avoid making work dependent on too many different PR. The only addition I can think of before doing it is the programming of the static mux to select the HWPE.
We should also take care of this PR https://github.com/pulp-platform/pulp-runtime/pull/45.

**Comment by ricted98** [comment]
@FrancescoConti is it possible that `neureka.h` is missing?

File: hwpe/neureka/src/nnx_layer.c [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nnx_layer.h"
+#include <pmsis.h>
+
+#include "neureka.h"
```
Comment (ricted98) [inline hwpe/neureka/src/nnx_layer.c:24]: @FrancescoConti  is it possible that `neureka.h` is missing?

Comment (FrancescoConti) [reply hwpe/neureka/src/nnx_layer.c:24]: Did you pull the submodule?

Comment (ricted98) [reply hwpe/neureka/src/nnx_layer.c:24]: You're right, my fault then!

**Pull request merged**